pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -28,21 +28,40 @@ type ServiceBindingRequestSpec struct {
 	// +optional
 	CustomEnvVar []corev1.EnvVar `json:"customEnvVar,omitempty"`
 
+	// NOTE: Services & Applications will be made non-optional
+	// after deprecated fields are removed.
+
+	// Services are used to identify multiple backing services.
+	// +optional
+	Services *[]BackingServiceSelector `json:"services,omitempty"`
```

Comment:
There's no need for the pointer there; see https://medium.com/rungo/the-anatomy-of-slices-in-go-6450e3bb2b94.

Reply:
This has been added to avoid `API rule violation: list_type_missing,github.com/redhat-developer/service-binding-operator/pkg/apis/apps/v1alpha1,ServiceBindingRequestSpec, Services ,` I believe

Reply:
@sbose78 I think it's worth removing this pointer, btw.

Reply:
@otaviof the change isn't really in this PR, the change is in https://github.com/redhat-developer/service-binding-operator/pull/369 .